dapp/src/components/dashboard/components/Sidebar/index.jsx

```diff
@@ -196,6 +196,14 @@ const Sidebar = ({ communityName, match, isAdmin, isGradientLogo, plugins, token
     return () => {}
   }, [isAdmin, tokenType])
 
+  useEffect(() => {
+    const paramsArr = location.pathname.split('/')
+    const lastItem = paramsArr[paramsArr.length - 1]
+    if ((communityAddress !== lastItem) && lastItem !== 'justCreated') {
```

Comment:
that something for analytics? You set it after the issuance?

Reply:
No, it's for setting the path in the initialization